### ISSUE TITLE
Creation of IoT Hub in Azure

### ISSUE THREAD

**Issue by MLopezJ** [MLopezJ/asset-tracker-cloud-coiote-azure-converter-js#2]
# TL;DR
Error was originated because of permissions restrictions. Permissions were update and creation of resource was possible.

# Issue Discussion
Following part 3 of  `Thingy:91` - `Coiote` integration [tutorial](https://iotdevzone.avsystem.com/docs/Demo_Projects/Tracking_tutorial/#:~:text=Coiote%20Device%20Center.-,Part%203%20%2D%20Connect%20Coiote%20to%20Microsoft%20Azure,-%23) I'm facing a blocker because I'm not able to create an IoT hub. 

## Error message

![Screenshot from 2023-02-07 13-09-14](https://user-images.githubusercontent.com/8351865/217541083-3d897090-6eb4-4906-86a5-f96cb9c1f134.png)

## Assumptions.
Subscription and resource group were selected just because were the options available on custom drop downs.

## Questions

What is my current permission?

What are the permissions needed?

**Comment by MLopezJ** [comment]
> What is my current permission?

I have `owner` permissions for `coiote-integration` resource group. 

![Screenshot from 2023-02-08 14-30-45](https://user-images.githubusercontent.com/8351865/217543939-e4277b52-15ad-4ba3-b7e4-8bc23aba19dd.png)

Even when I'm trying to create a resource into the `Resource Group` I own, I received the same initial error message. 

What is a `resource group` in azure?

**Comment by MLopezJ** [comment]
> What are the permissions needed?

In order with the error message, I need `Devices` and `Security` on my `nRF Asset Tracker (Coiote Integration)` subscription. But right now I don't know if that makes sense. I mean, I can not confirm if everything I need to do is request `Devices` and `Security` access on my subscription. 

I also can not confirm what is my role on the subscription, because I don't have the permissions to read it. 

![Screenshot from 2023-02-08 14-42-55](https://user-images.githubusercontent.com/8351865/217546716-ea765162-4729-4f39-b4c1-b6c89ece0bb4.png)

**Comment by MLopezJ** [comment]
> What is a `resource group` in azure?

Long story short; a `resource group` is an environment. 

It is a logical collections of virtual machines, storage accounts, virtual networks, web apps, databases, and/or database servers. Can be used to group related resources. 

![image](https://user-images.githubusercontent.com/8351865/217547986-0cab13e4-8163-42cc-82d6-1c12b8bc2e6d.png)

**Comment by MLopezJ** [comment]
> In order with the error message, I need `Devices` and `Security` on my `nRF Asset Tracker (Coiote Integration)` subscription. But right now I don't know if that makes sense. I mean, I can not confirm if everything I need to do is request `Devices` and `Security` access on my subscription.
> 
> I also can not confirm what is my role on the subscription, because I don't have the permissions to read it.

Subscription permissions updated. 

`IoT hub` created successfully.

**Comment by MLopezJ** [comment]
> In order with the error message, I need `Devices` and `Security` on my `nRF Asset Tracker (Coiote Integration)` subscription. But right now I don't know if that makes sense. I mean, I can not confirm if everything I need to do is request `Devices` and `Security` access on my subscription.
> 
> I also can not confirm what is my role on the subscription, because I don't have the permissions to read it.

Subscription permissions updated. 

`IoT hub` created successfully.
